### DISCUSSION TITLE
Custom theme Pages is not rendering in GHE

### DISCUSSION THREAD

**Discussion by shivajipotnuru** [community/community#47123] [Enterprise]
### Select Topic Area

Bug

### Body

I have used "just-the-docs" theme in my repo & the solution is working perfectly fine in my local machine. when i published to the GHE branch and pages source also correctly mapped, the page is not rendering properly.

Requesting your help on this issue. I have posted the same issues in stack Overflow. Kindly please help me to resolve this issue.

https://stackoverflow.com/questions/75421760/problem-with-github-pages-using-remote-theme-in-github-enterprise

**Comment by jge162** [comment]
There is not enough information to assist you. There are a large amount of variables that could be causing this issue. 

Some basic things to check would be -> Double-check that you have uploaded all the necessary files for the "just-the-docs" theme to your GHE instance. This includes the `assets, _data`,` _layouts`, `_sass`, and includes directories, as well as the `README.md` and `LICENSE.txt` files. Make sure they are in the correct locations in your GHE repository.

Not much help I know but, need a lot more context. Sorry

**Reply by shivajipotnuru** [reply]
Hi @jge162 , Thanks for answering the issue, I have cloned the entire 'just-my-docs' public repo to new repository in GHE. In my actual documentation repo inside _config.yml I have mapped with relative path still it was not able to fetch it.

second option with remote_theme property which maps to public hosted 'jut-mydocs' repo URL, this one is not even rendering the webpage.
